audiomentations/augmentations/padding.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+
+class Padding(BaseWaveformTransform):
+    """
+    Apply padding to the audio signal -  take a fraction of the end or the start of the audio
+    and treat that part as padding.
+    """
+    
+    supports_multichannel = True
+    
+    def __init__(self, mode='constant', p=0.5):
+        super().__init__(p)
+        self.mode = mode
+
+    def apply(self, samples, sample_rate):
+        orig_len = len(samples)
+        skip_idx = np.random.randint(1, orig_len-1)
+        r = np.random.random()
+        if r < 0.5:
+            samples = samples[:skip_idx]
```

Comment:
```suggestion
            samples = samples[..., :skip_idx]
```

for multichannel support

Reply:
Also done